libmozdata/config.py

```diff
@@ -48,7 +59,9 @@ class ConfigEnv(Config):
     def get(self, section, option, default=None, type=str):
         env = os.environ.get("LIBMOZDATA_CFG_" + section.upper() + "_" + option.upper())
         if not env:
-            return default
+            if default is not None:
+                return default
+            return super().get(section, option)
```

Comment:
Same here
```suggestion
            return default or super().get(section, option)
```

Reply:
Apparently I have to keep the condition "is not None" because the default value can be an empty string, like here

```
class Connection(object):
    """Represents a connection to a server"""

    VERIFY_SSL = True
    TIMEOUT = 30
    MAX_RETRIES = 256
    MAX_WORKERS = multiprocessing.cpu_count()
    RAISE_ERROR = True
    CHUNK_SIZE = 32
    TOKEN = ""
    USER_AGENT = None
    X_FORWARDED_FOR = utils.get_x_fwed_for_str(
        config.get("X-Forwarded-For", "data", "")
    )
```

Reply:
nit: Since we need to keep the `is not None`, having it like before would be better for readability.